### PR TITLE
Issues 62 63 65

### LINKED ISSUE #62: On /home, swap "Search cases" and "Browse all" for cases
Swap them so that it matches the Guide functionality.

### PR DESCRIPTION
**Pull request by musangowope**
- Add Laws.Africa logo to "About this app" section on /home
- Change red accent colour to blue
- On /home, swap "Search cases" and "Browse all" for cases

resolves #62 #63 #65

### CODE REVIEW

File: src/pages/home/Home.tsx [hunk 1]
```diff
@@ -75,7 +74,7 @@ const Home: React.FC = () => {
               <h5 className="section-heading">Cases</h5>
             </IonCol>
             <IonCol size="1" className="ion-text-nowrap">
-              <Link className="ion-float-right link-plain" to="/cases/">Browse All</Link>
+              <Link className="ion-float-right link-plain" to="/cases/">Search</Link>
```
Comment (longhotsummer) [inline src/pages/home/Home.tsx:77]: the link targets must change too.

File: src/pages/home/Home.tsx [hunk 2]
```diff
@@ -108,42 +107,66 @@ const Home: React.FC = () => {
...
-                onClick={() => handleSupportersLink('https://africanlii.org/')}
-              />
+              <a href="https://africanlii.org/"
+                 target="_blank"
+                 rel="noopener noreferrer"
+              >
+                <img
+                    alt=""
+                    src="/assets/African Lii logo.svg"
+                    onError={(e) => {
```
Comment (longhotsummer) [inline src/pages/home/Home.tsx:121]: this onError stuff is a hold over and really isn't necessary, let's start doing away with it.

**Review by longhotsummer — CHANGES_REQUESTED** [review]
Looks good, just the URLs for the cases.

**Comment by longhotsummer** [comment]
@musangowope oh, pls also take a new screenshot for use on the landing page now that colours have changed. I did it before with chrome:

1. responsive mode
2. ipad in landscape mode
3. turn "show device frame" on in chrome responsive mode (three dots top right corner)

File: src/pages/home/Home.tsx (hunk 1 above)
Comment (musangowope) [reply src/pages/home/Home.tsx]: done

File: src/pages/home/Home.tsx (hunk 2 above)
Comment (musangowope) [reply src/pages/home/Home.tsx]: removed

**Comment by musangowope** [comment]
@longhotsummer 
did a recording instead of screenshot demoing new colors and responsiveness for landing page
https://www.loom.com/share/801f5962b663499692d3af4de78a8e7d

**Comment by musangowope** [comment]
@longhotsummer update the phone and tablet assets with the new colors

**Review by longhotsummer — APPROVED** [review]
👌🏻

**Pull request merged**